[2024-06-30 19:01:19,232] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T15:00:00+00:00 [queued]>
[2024-06-30 19:01:19,924] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T15:00:00+00:00 [queued]>
[2024-06-30 19:01:20,127] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:20,136] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-06-30 19:01:20,180] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:20,420] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_datalake_table> on 2024-06-30 15:00:00+00:00
[2024-06-30 19:01:20,477] {standard_task_runner.py:52} INFO - Started process 1330 to run task
[2024-06-30 19:01:20,544] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_datalake_dag', 'create_datalake_table', 'scheduled__2024-06-30T15:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp5h701uoa', '--error-file', '/tmp/tmpkahk25sw']
[2024-06-30 19:01:20,583] {standard_task_runner.py:80} INFO - Job 24: Subtask create_datalake_table
[2024-06-30 19:01:22,204] {task_command.py:371} INFO - Running <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T15:00:00+00:00 [running]> on host 12d6cc628018
[2024-06-30 19:01:22,956] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_datalake_dag
AIRFLOW_CTX_TASK_ID=create_datalake_table
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-30T15:00:00+00:00
[2024-06-30 19:01:23,169] {base.py:68} INFO - Using connection ID 'datalake_postgres' for task execution.
[2024-06-30 19:01:23,229] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS datalake_table (
            id SERIAL PRIMARY KEY,
            data VARCHAR(50)
        );
        , parameters: None
[2024-06-30 19:01:23,255] {postgres.py:94} INFO - NOTICE:  relation "datalake_table" already exists, skipping

[2024-06-30 19:01:23,381] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_datalake_dag, task_id=create_datalake_table, execution_date=20240630T150000, start_date=20240630T190119, end_date=20240630T190123
[2024-06-30 19:01:23,563] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-30 19:01:24,933] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
